hockeysdk/src/main/java/net/hockeyapp/android/ExceptionHandler.java

```diff
@@ -218,7 +217,7 @@ private static void writeValueToFile(String value, String filename) throws IOExc
                 writer.flush();
             }
         } catch (IOException e) {
-            // TODO: Handle exception here
+            e.printStackTrace();
```

Comment:
[Cause it used widely here.](https://github.com/bitstadium/HockeySDK-Android/search?utf8=%E2%9C%93&q=printStackTrace&type=) Should I change all entries?